metashare/api/sf_org_changes.py

```diff
@@ -13,7 +13,60 @@
 from .sf_run_flow import refresh_access_token
 
 
-def run_retrieve_task(user, scratch_org, project_path, desired_changes):
+def get_valid_target_directories(scratch_org_config):
+    """
+    Expects to be called from within a `local_github_checkout`.
+    """
+    package_directories = {}
+    if scratch_org_config.get("source_format") == "sfdx":
+        with open("sfdx-project.json") as f:
```

Comment:
Do we want a fallback e.g. `if os.path.exists("sfdx-project.json")`?

Reply:
Not until we sort out the question above.

Reply:
Okay... Even if it is set as `source_format: "sfdx"`, if it does not have a `sfdx-project.json` file then I assume we should fall back to `src` as the default. What else would you propose -- raising an error?

Reply:
That would be an error condition, yeah -- it would just error out when trying to deploy to an org if source_format is sfdx but there's no sfdx-project.json